### DISCUSSION TITLE
整合 spring boot 适合吗

### DISCUSSION THREAD

**Discussion by chenjunwenhao** [apache/doris#38775] [A - General / Q&A]
大数据的可视化场景下。传统数据库存储空间和查询性能都有点捉襟见肘了。doris适合做这方面吗？结果数据放到doris，然后spring boot连doris对外提供服务。doris适合做这种场景吗，还是使用其他中间件更合适一些比如es或者其他中间件？

**Comment by maruyue** [comment]
可以去ask.selectdb.com中文论坛问问题。这里更多是英文论坛。
doris就是实时分析数据库，支持分析可视化，至于是不是sprintboot这个没关系，你用什么框架和doris没有关系的。
